spec/features/ncr/work_orders/create_spec.rb

```diff
@@ -104,6 +100,22 @@
       expect(page).to have_content('BA80')
     end
 
+    scenario "does not show system approver emails as approver options", :js do
+      expect(Ncr::WorkOrder.all_system_approver_emails.size).to eq 4
+      login_as(requester)
+      approving_official = create(:user)
+      visit "/ncr/work_orders/new"
+      within(".ncr_work_order_approving_official_email") do
+        find(".selectize-control").click
+      end
+
+      expect(page).to have_content(approving_official.email_address)
+      expect(page).not_to have_content(Ncr::WorkOrder.ba61_tier1_budget_mailbox)
+      expect(page).not_to have_content(Ncr::WorkOrder.ba61_tier2_budget_mailbox)
+      expect(page).not_to have_content(Ncr::WorkOrder.ba80_budget_mailbox)
+      expect(page).not_to have_content(Ncr::WorkOrder.ool_ba80_budget_mailbox)
```

Comment:
Can these assertions focus on the select control itself rather than the whole page?